tests/misc/tmissingnilcheck.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-  disabled: true
+    disabled: true
```

Comment:
Created disabled around 2014-04 in a commit that references #1075

Reply:
How does it fail?

Reply:
```
Expected:
exitcode: 0

Gotten:
exitcode: 1

Output:
Traceback (most recent call last)
tmissingnilcheck.nim(20) tmissingnilcheck
tmissingnilcheck.nim(18) newConnection
gc.nim(280)              asgnRef
SIGSEGV: Illegal storage access. (Attempt to read from nil?)

```

Reply:
I really don't get how this is a missing nil check test... weird.

unless the history provides some clues I would consider deleting it.

Reply:
None whatsover. No meaningful changes since its inception.

Reply:
Ick, let's delete it.